src/screens/Topics/ListTopics.js

```diff
@@ -8,7 +8,7 @@ const ListTopics = ({item, i, myTopic, handleSelectedLanguage}) => (
     key={i}
     style={[
       styles.bgTopicSelectNotActive,
-      {backgroundColor: myTopic[item.topic_id] ? colors.bondi_blue : colors.concrete}
+      {backgroundColor: myTopic[item.topic_id] ? colors.signed_primary : colors.concrete}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `colors.bondi_blue` to `colors.signed_primary` seems to be a color scheme update. Ensure that this new color (`colors.signed_primary`) is defined in your colors utility module and it aligns with the overall app's color scheme.
```diff
- {backgroundColor: myTopic[item.topic_id] ? colors.bondi_blue : colors.concrete}
+ {backgroundColor: myTopic[item.topic_id] ? colors.signed_primary : colors.concrete}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->